docs/UserGuide.md

```diff
@@ -94,10 +94,9 @@ DESCRIPTION OF PARAMETER
 
 
 #### `KEYWORD`
-DESCRIPTION OF PARAMETER
-* FORMAT AND RESTRICTIONS WITH JUSTIFICATION
-* (if applicable) For best usage, ...
-* (if applicable) Valid examples (if not clear from above)
+Keyword used in commands such as find
```

Comment:
Hmm, just in case, maybe make this "A keyword used in the various find commands."

Just in case someone points out that there is no `find` command.

Reply:
Fixed